.github/workflows/bump-deps.yaml

```diff
@@ -0,0 +1,65 @@
+name: Bump DEPS
+on:
+  workflow_dispatch:
+    inputs:
+      dep:
+        description: "Dependency"
+        required: true
+        type: string
+        default: "vendor/web-discovery-project"
+      ref:
+        description: "New ref for dependency"
+        required: true
+        type: string
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  auto-bump:
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: "0"
+
+      - name: auto-bump
+        run: |
+          set -x
+
+          git config user.email "${{ github.actor }}@users.noreply.github.com"
+          git config user.name "${{ github.actor }}"
+          git config push.default simple
+          gh api /repos/brave/web-discovery-project/commits/main -q .sha
+          exit 0
+
+          SED=sed
+          DATE=date
+
+          # Inputs
+          DEP="${{ inputs.dep }}"
+          DEPREF="${{ inputs.ref }}"
+          [[ -z "$DEP" ]] && { echo Input dependency cannot be empty; exit 1; }
+          [[ -z "$DEPREF" ]] && { echo Input ref cannot be empty; exit 1; }
+
+          DATETIME="$($DATE '+%Y-%m-%d-%H-%M')"
+          readarray -t milestones < <(gh api /repos/brave/brave-core/milestones -q ".[] | .title" | grep -Fv Closed)
+          branches=( "${milestones[@]/*Nightly/master}" )
+          branches=( "${branches[@]/ -*/}" )
+          echo Branches to operate on:
+          declare -p branches
```

Comment:
This can just be `echo "Branches to operate on: ${branches[*]}"`

Reply:
Done